api/tests/mocha/services/connected-user-log.spec.js

```diff
@@ -0,0 +1,83 @@
+const chai = require('chai');
+const sinon = require('sinon');
+const moment = require('moment');
+const connectedUserLogService = require('../../../src/services/connected-user-log');
+const db = require('../../../src/db');
+
+describe('Connected Users Log service', () => {
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  describe('saveLog()', () => {
+    it('should save a log', () => {
+      const getStub = sinon.stub(db.medicLogs, 'get').returns(Promise.reject({ status: 404 }));
```

Comment:
Thanks for covering this case. I saw it when reading the implementation code and immediately scrolled down to check for a test for it :)